ui/app/helpers/conversions.util.js

```diff
@@ -7,31 +8,44 @@ export function hexToDecimal (hexValue) {
   })
 }
 
-export function getEthFromWeiHex ({
-  value,
-  conversionRate,
-}) {
-  return getValueFromWeiHex({
-    value,
-    conversionRate,
-    toCurrency: 'ETH',
-    numberOfDecimals: 6,
-  })
+export function getEthConversionFromWeiHex ({ value, conversionRate, numberOfDecimals = 6 }) {
+  const denominations = [ETH, GWEI, WEI]
```

Comment:
If I am reading this right, it is to ensure that the activity log shows the value associated with each activity in the largest possible denomination. Is that correct?

Reply:
Right, or another way of putting it, if the value is too small for a particular denomination (and the result is 0 ETH, for example), it goes to a lower denomination and so on.

Reply:
I think for the "transaction created" row, the value should always be shown in ETH. When the user creates the transaction, they will be creating it in ETH, so showing in GWEI or WEI is confusing.

![screenshot from 2018-09-06 15-14-45](https://user-images.githubusercontent.com/7499938/45175331-cb3af300-b1e7-11e8-8a5f-1d3880b00802.png)